src/components/App/App.tsx

```diff
@@ -1,8 +1,60 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Route, Switch, NavLink } from 'react-router-dom';
 import { Header } from '../Header/Header';
+import mockPosts from '../../mockdata/mockPosts.json';
+import { Cards } from '../Cards/Cards';
 
-interface AppProps {}
+export interface Post {
+  type: string;
+  id: number;
+  attributes: {
+    user_id: number;
+    username: string;
```

Comment:
Also, username won't be stored at the same level on a post -- again, I am not super clear on how this will connect but in graph it'll look something like this:

```
posts {
    id
    title
    description
    user {
        id
        username
    }
}
```

(obviously with more info there and all)

Reply:
Made some changes to the type to reflect this, but may need to change more once I hook up to graphQL